packages/playground/src/App.vue

```diff
@@ -212,8 +252,8 @@ const $router = useRouter();
 const profileManager = useProfileManager();
 const gridStore = useGrid();
 const network = process.env.NETWORK || (window as any).env.NETWORK;
-
-const openProfile = ref(false);
+const toolbarExtended = ref(false);
+const openProfile = ref(true);
```

Comment:
@0oM4R openProfile was set to `false` in your fix right? so we still need it or it can be `true` now?

Reply:
it should be initialized with false yes

Reply:
@MohamedElmdary @0oM4R should I change something here at my PR?

Reply:
@ehab-hassan @0oM4R let's merge it now as it is and later invest more on how to fix that issue in another branch